php.ini-development

```diff
@@ -1249,7 +1254,7 @@ pgsql.max_links = -1
 pgsql.ignore_notice = 0
 
 ; Log PostgreSQL backends Notice message or not.
-; Unless pgsql.ignore_notice=0, module cannot log notice message.
+; Unless pgsql.ignore_notice = 0, module cannot log notice message.
 ; https://php.net/pgsql.log-notice
 pgsql.log_notice = 0
```

Comment:
Boolean INI setting?

Reply:
According to the [documentation](https://www.php.net/manual/en/pgsql.configuration.php#ini.pgsql.ignore-notice) this is int.

Reply:
Ditto